src/Game.cpp

```diff
@@ -121,4 +121,28 @@ void Game::setValueFrameAndPrevFrames(int pos, const std::string& value)
             frame_[pos - 2].addValue(firstBall);
         }
     }
+    currentRolls_++;
+    if (pos == 9 && value.size() == 2 && value[1] == '/')
+    {
+        numberRolls_ = 11;
+    }
+    if (pos == 9 && value[0] == 'X')
+    {
+        numberRolls_ = 12;
+    }
+    if (pos == 9 && value.size() == 1)
+    {
+        currentRolls_ = 9;
+    }
+}
+std::string Game::getStatus()
+{
+    if (currentRolls_ == numberRolls_)
+    {
+        return "Game finished.";
+    }
+    else
+    {
+        return "Game in progress.";
```

Comment:
In requirements is `game in progress` `no game` `game finished`. Please change the formatting of this sentence.